packages/patternfly-4/react-core/src/experimental/components/DataToolbar/DataToolbarItem.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DataToolbar/data-toolbar';
+import { css, getModifier } from '@patternfly/react-styles';
+
+import {
+  DataToolbarBreakpointMod,
+  DataToolbarSpacer,
+  formatBreakpointMods,
+  formatItemSpacers
+} from './DataToolbarUtils';
+
+export enum DataToolbarItemMod {
```

Comment:
I added clear, but the shared-item modifier and chip-group are going to be used in the expandable content - and (probably) only used internally, applied programmatically. When the chip group is implemented soon it should be clear.

Reply:
and actually i just got clarification - clear is only intended to be used alongside multiple chip groups in the expandable content so it will not be provided by the consumer. so i think the existing variants are appropriate.

Reply:
`clear`, `shared-item` and `chip-group` are dynamically added to toolbar. 

- `clear`: when multiple filters have been selected
- `shared-item`: this will probably go as React doesn't need it
- `chip-group`: automatically added when a filter is selected